src/actors/builtin/storage_miner/storage_miner_actor_code.go

```diff
@@ -665,16 +676,16 @@ func (a *StorageMinerActorCode_I) _rtVerifySealOrAbort(rt Runtime, onChainInfo s
 	pieceInfos, err := sector.Deserialize_PieceInfos(rt.SendQuery(
 		addr.StorageMarketActorAddr,
 		ai.Method_StorageMarketActor_GetPieceInfosForDealIDs,
-		[]util.Serialization{
-			sector.Serialize_SectorSize(sectorSize),
-			deal.Serialize_DealIDs(onChainInfo.DealIDs()),
-		},
+		serde.MustSerializeParams(
+			sectorSize,
+			onChainInfo.DealIDs(),
+		),
 	))
 	Assert(err == nil)
 
 	// Unless we enforce a minimum padding amount, this totalPieceSize calculation can be removed.
 	// Leaving for now until that decision is entirely finalized.
-	var totalPieceSize util.UInt
+	var totalPieceSize uint64
```

Comment:
We have removed the maximum padding amount enforcement, unclear what minimum padding here refers to (could be a typo). totalPieceSize is not used anywhere, we can remove this calculation.

Reply:
Thanks, I'll save it for another PR.